tests/tests_provider_nm.yml

```diff
@@ -5,21 +5,21 @@
 - hosts: all
   name: Run playbook 'playbooks/tests_provider.yml' with nm as provider
   tasks:
-    - include_tasks: tasks/el_repo_setup.yml
+    - include_tasks: tasks/el_repo_setup.yml  # noqa name
     - name: Set network provider to 'nm'
       set_fact:
         network_provider: nm
       tags:
         - always
 
-    - block:
+    - block:  # noqa name
         - name: Install NetworkManager
           package:
             name: NetworkManager
             state: present
-        - name: Get NetworkManager version
+        - name: Get NetworkManager version  # noqa command-instead-of-module
           command: rpm -q --qf "%{version}" NetworkManager
-          register: NetworkManager_version
+          register: networkmanager_version
```

Comment:
where is this used?

Reply:
It is used in - networkmanager_version.stdout is version({minimum_nm_version}, '>=').

Reply:
ok - because https://github.com/linux-system-roles/network/blob/54c556c06096eb29ce6a97a1f3d07fd17609a5e3/tests/tests_provider_nm.yml#L37 is still using `NetworkManager_version`